pkg/deploy/lattice/access_log_subscription_synthesizer.go

```diff
@@ -49,8 +49,12 @@ func (s *accessLogSubscriptionSynthesizer) Synthesize(ctx context.Context) error
 			als.Status = alsStatus
 		case core.UpdateEvent:
 			s.log.Debugf("Started updating Access Log Subscription %s", als.ID())
-			// TODO
-			return nil
+			alsStatus, err := s.accessLogSubscriptionManager.Update(ctx, als)
+			if err != nil {
+				return err
+			} else {
+				als.Status = alsStatus
```

Comment:
Just noticed this could be denested, will fix next commit alongside any other feedback